src/components/TechTab.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <b-container fluid class="tech-tab">
+    <b-table :items="allTechs" :fields="fields" thead-class="d-none" tbody-tr-class="tech-row">
+      <template #cell(tech)="data">
+        <div class="tech-name text-right">
+          <b-badge v-if="data.item.advanced" variant="danger">A</b-badge>
+          {{ techTitle(data.item) }} <b-badge :variant="techBadgeVariant(data.item)">{{ data.item.currentLevel }}</b-badge>
+        </div>
+      </template>
+
+      <template #cell(buy)="data">
+        <b-button variant="primary"
+                  size="sm"
+                  v-on:click="increaseTechnology(data.item, false)"
+                  v-bind:disabled="disableBuy(data.item)">
+          Buy
+        </b-button>
+      </template>
+
+      <template #cell(wreck)="data">
+        <b-button v-if="data.item.wreck" 
+                  variant="primary"
+                  size="sm"
+                  v-on:click="increaseTechnology(data.item, true)"
+                  v-bind:disabled="disableWreck(data.item)">
+          Wreck
+        </b-button>
+      </template>
+
+      <template #cell(costs)="data">
+        <b-list-group horizontal>
+          <b-list-group-item 
+              v-for="(cost, level) in techCosts(data.item)"
+              v-bind:variant="techLevelVariant(data.item, level)"
+              v-bind:key="(cost, level)">
+            <strong>{{ level }}</strong> <small>({{ cost }})</small>
+          </b-list-group-item>
+        </b-list-group>
+      </template>
```

Comment:
These level info buttons are too big for some mobile devices and a horizontal scroll appears. Maybe we can reduce the size.

Reply:
I'll take a look.

Reply:
How small of a screen?  I was testing on a screen with a width of 360px.  What would the best "minimum" be?

Reply:
Hmm.  I am testing it in Chrome's Developer Tools, which lets you test it out in "responsive" mode with whatever screen widths you want, and even as small as 280px width it's not doing a horizontal scroll for me.  Granted, it may be different on an actual device, though.